docs/gatehub.md

```diff
@@ -0,0 +1,73 @@
+---
+id: gatehub
+title: GateHub
+sidebar_label: GateHub
+---
+
+Use GateHub to have your earnings paid out in XRP or any other supported
+currency.
+
+> You can see a list of supported currencies on [GateHub's support site](https://support.gatehub.net/hc/en-us/articles/360021426493-Supported-currencies).
+
+## Sign up
+
+1. Go to Gatehub.net and [sign up](https://signin.gatehub.net/signup) for an account.
+2. Complete GateHub's account verification process.
+
+> If you need help, check out their articles:
+> * [Register a GateHub Account](https://support.gatehub.net/hc/en-us/articles/360021318533-Register-a-GateHub-Account)
+> * [Verification process for an individual account](https://support.gatehub.net/hc/en-us/articles/360021131054-Verification-process-for-an-individual-account)
+
+After your account is verified you can create your wallet on GateHub.
+
+## Create your wallet
+
+1. Log in to your [GateHub account](https://signin.gatehub.net).
+2. Select **Wallet** from the left navigation bar.
+3. Click **\(+\)** in the upper-left.
+4. Enter your password.
+5. Enter a wallet name and select **Hosted** under **Wallet type**.
+6. Click **Create**. Your wallet is now created on GateHub.
+7. Click **Deposit/Receive** at the bottom of the screen.
+8. Select the **ILP** tab and copy your payment pointer to your clipboard.
+
+![createwallet](assets/gatehub-01.png)
+
+## Create your meta tag
```

Comment:
We may not want to describe this here. A lot of the time people will be sending their creators to this guide so they can get their payment pointer and enter it into a website (like cinnamon or into a wordpress plugin) rather than using the meta tag directly.

Maybe instead we can tell the user they can return to the site that needs their payment pointer and enter it, or add it to their own site by following the getting started tutorial?

Reply:
Updated the three wallet-specific pages.